.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
 		"**/.pnp.*": true
 	},
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
```

Comment:
uhh

Reply:
i'm not too sure about that one either. i'll fix it tomorrow

Reply:
It's a VSC/extension update that changed the default setting, we have to update it here too anyways, because every time you restart VSC it will update this setting